include/envoy/network/address.h

```diff
@@ -125,5 +125,104 @@ class Instance {
 
 typedef std::shared_ptr<const Instance> InstancePtr;
 
+/*
+ * Parse an internet host address (IPv4 or IPv6) and create an Instance from it.
+ * The address must not include a port number.
+ * @param ipAddr string to be parsed as an internet address.
+ * @return pointer to the Instance, or nullptr if unable to parse the address.
+ */
+InstancePtr parseInternetAddress(const std::string& ipAddr);
```

Comment:
Regarding parseInternetAddress, I've moved it (locally) into Utility in utility.h, but realize I'm heading towards a dependency cycle. I'll be modifying utility.h later to use CidrRange in IpList (or a newly named class), and CidrRange depends on parseInternetAddress. Sigh. I'll come up with something.